### ISSUE TITLE
Online Resource Link Field

### ISSUE THREAD

**Issue by mguarinello** [northwest-knowledge-network/mdedit#37]
The field should be call "Online Resources" 
The instructions should be "Is your data or any additional information specific to the data available online or is your data provided as a web service (e.g., Web Mapping Services, Catalog Services for the web, REST services). Provide the url for each online resource."
This should be at the bottom of the Access Contacts block and should be enabled to add more than one.

**Comment by mtpain** [comment]
Just to be clear, the online resource links should be separate from any individual access contact, right?

**Comment by mguarinello** [comment]
Sorry. I know that one is confusing. The online resource link is actually within the <distributor> tags in the ISO schema. We need to have the option to add more than one online link per distributor.


Marisa Guarinello
Environmental Data Manager
MILES EPSCoR Project
Northwest Knowledge Network
University of Idaho, Office of Research & Economic Development
mguarinello@uidaho.edu
208-885-5044

**Comment by mguarinello** [comment]
We need to revisit this one with input from Ed. He said online resource can either be located with the access contact or separately. I need to check out the schema and we all need to figure out which method or both we want to use. Sorry for all the confusion on this one!

**Comment by mguarinello** [comment]
Okay. I've looked into this and here is what to use on the next iteration. This is for data format and online resources. #30 

1) Make a new info 'card' for Distribution Info
1) Add Data format as a separate field, dropdown with multiple entry and write in
1) Code as /root/record/data_format/item and then have each entry as 'word', like what we did for keywords
1) Add a write-in field for compression technique, single entry This should be an optional field. Code as /root/record/compress
1) Add Online Resources as a separate field. Allow multiple entries. This should be an optional field.
1) Code as /root/record/online/item and then each entry as "url", similar to what we did for keywords, just using 'url' instead of 'word'

Note: under Distribution, distributor contact is separate tab group from distributor format and online, just confusing b/c ‘distributor’ not ‘distribution’
